crypto-core/src/main/java/com/palantir/crypto2/io/ApacheCtrDecryptingSeekableInput.java

```diff
@@ -54,7 +54,7 @@ public static ApacheCtrDecryptingSeekableInput create(SeekableInput input, KeyMa
     /**
      * Returns true if the OpenSSL is able to be loaded.
      */
-    public boolean isSupported() {
+    public static boolean isSupported() {
```

Comment:
it'll fail when you try to instantiate it. the key part being the fact that I explicitly pass in properties that force it to use OpenSSL and fail otherwise: https://github.com/palantir/hadoop-crypto/blob/develop/crypto-core/src/main/java/com/palantir/crypto2/io/ApacheCtrDecryptingSeekableInput.java#L129-L133